fileutils.opam

```diff
@@ -5,22 +5,17 @@ homepage: "http://ocaml-fileutils.forge.ocamlcore.org/"
 license: "LGPL-2.1 with OCaml linking exception"
 dev-repo: "git+https://github.com/gildor478/ocaml-fileutils.git"
 bug-reports: "https://github.com/gildor478/ocaml-fileutils/issues"
+
 build: [
-  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
-  ["ocaml" "setup.ml" "-configure" "--enable-tests"] {with-test}
-  ["ocaml" "setup.ml" "-build"]
-  ["ocaml" "setup.ml" "-doc"] {with-doc}
-  ["ocaml" "setup.ml" "-test"] {with-test}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
```

Comment:
Can you add 
["dune" "build" "@doc" "-p" name] {with-doc}

Reply:
(done, but I think you tagged the person who has the pseudo `doc` on GitHub :p)